pkg/cli/admin/upgrade/recommend/recommend.go

```diff
@@ -142,6 +149,36 @@ func (o *options) Run(ctx context.Context) error {
 		fmt.Fprintf(o.Out, "info: An update is in progress.  You may wish to let this update complete before requesting a new update.\n  %s\n", strings.ReplaceAll(c.Message, "\n", "\n  "))
 	}
 
+	if o.precheckEnabled {
+		conditions, err := o.precheck(ctx)
+		if err != nil {
+			fmt.Fprintf(o.Out, "Failed to check for at least some preconditions: %v\n", err)
+		}
+		var happyConditions []string
+		var unhappyConditions []string
+		for _, condition := range conditions {
+			if condition.Status == metav1.ConditionTrue {
+				happyConditions = append(happyConditions, fmt.Sprintf("%s (%s)", condition.Type, condition.Reason))
+			} else {
+				unhappyConditions = append(unhappyConditions, condition.Type)
+			}
+		}
+		if len(happyConditions) > 0 {
+			sort.Strings(happyConditions)
+			fmt.Fprintf(o.Out, "The following conditions found no cause for concern in updating this cluster to later releases: %s\n\n", strings.Join(happyConditions, ", "))
+		}
+		if len(unhappyConditions) > 0 {
+			sort.Strings(unhappyConditions)
+			fmt.Fprintf(o.Out, "The following conditions found cause for concern in updating this cluster to later releases: %s\n\n", strings.Join(unhappyConditions, ", "))
+
+			for _, c := range conditions {
```

Comment:
nit: it'd be a little friendlier if the condition order matched between the summary line on L172 and detailed list but no big deal and we can polish that later if we want

Reply:
hmm, yeah, the current condition response is roughly sorted in order of decreasing severity, which is fine as long as we only have the `alerts` prechecker, but won't scale if/when we add other precheckers.  We can add some kind of structured severity declaration to our conditions, but yeah, I'll defer figuring that out until future work.